lib/modules/datasource/dotnet/schema.ts

```diff
@@ -0,0 +1,51 @@
+import { z } from 'zod';
+
+const Product = z.union([z.literal('.NET Core'), z.literal('.NET')]);
```

Comment:
should we name those cont with `Schema` suffix and remove the `Type` suffix from types?

🤔

Reply:
Yeah, can do. I think this is the first datasource to use JSON schemas, so I am still figuring out the best practices.